src/forklift/models.py

```diff
@@ -321,14 +322,17 @@ def __init__(
 
         if '.sde' in self.source.lower():
             self._try_to_find_data_source_by_name()
-        if not arcpy.Exists(self.source):
-            self.result = (Crate.INVALID_DATA, 'source data is inaccessible')
-            return
 
         try:
-            self.source_describe = describer(self.source)
+            if self.source.lower() in describes_cache:
```

Comment:
**question**: is source the best item to match on? Will you duplicate describes with two different `.sde` file names?

Reply:
Yes, it would duplicate on different `.sde` file names. Although, I believe the vast majority are using the `SGID.sde` connection in the garage. I'm open to other ideas on properties to cache on. I wasn't comfortable just matching the feature class name. But maybe that's safe?

Reply:
if we are consistent on using the garage then I am ok with this.